modules/configuring-the-lora-training-job.adoc

```diff
@@ -0,0 +1,249 @@
+
+
+.Prerequisites
+ifdef::upstream,self-managed[]
+* You have logged in to {openshift-platform}.
+endif::[]
+ifdef::cloud-service[]
+* You have logged in to OpenShift.
+endif::[]
+
+ifndef::upstream[]
+* You have access to a data science cluster that is configured to run distributed workloads as described in link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
+endif::[]
+ifdef::upstream[]
+* You have access to a data science cluster that is configured to run distributed workloads as described in link:{odhdocshome}/working-with-distributed-workloads/#configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
+endif::[]
+
+ifndef::upstream[]
+* You have created a data science project. 
+For information about how to create a project, see link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/using-data-science-projects_projects#creating-a-data-science-project_projects[Creating a data science project].
+endif::[]
+ifdef::upstream[]
+* You have created a data science project. 
+For information about how to create a project, see link:{odhdocshome}/working-on-data-science-projects/#creating-a-data-science-project_projects[Creating a data science project].
+endif::[]
+
+* You have Admin access for the data science project.
+** If you created the project, you automatically have Admin access. 
+** If you did not create the project, your cluster administrator must give you Admin access.
+
+ifndef::upstream[]
+* If Kueue is not installed in your environment, you have configured the Training Operator permissions as described in link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/tuning-a-model-by-using-the-training-operator_distributed-workloads/#configuring-the-training-operator-permissions-when-not-using-kueue_distributed-workloads[Configuring the Training Operator permissions when not using Kueue].
+endif::[]
+ifdef::upstream[]
+* If Kueue is not installed in your environment, you have configured the Training Operator permissions as described in link:{odhdocshome}/working-with-distributed-workloads/#configuring-the-training-operator-permissions-when-not-using-kueue_distributed-workloads[Configuring the Training Operator permissions when not using Kueue].
+endif::[]
+
+* You have access to a model.
+* You have access to data that you can use to train the model.
+
+.Procedure
+. In a terminal window, if you are not already logged in to your OpenShift cluster, log in to the OpenShift CLI as shown in the following example:
++
+[source,subs="+quotes"]
+----
+$ oc login __<openshift_cluster_url>__ -u __<username>__ -p __<password>__
+----
+
+. Configure a LoRA training job, as follows:
```

Comment:
A nitpick - the ConfigMap is not a training job, it is a configuration for training job.